backend/canisters/group/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Simplify `c2c_update_proposals` ([#3621](https://github.com/open-ic/open-chat/pull/3621))
 - Move `delete_messages` into `group_chat_core` ([#3622](https://github.com/open-ic/open-chat/pull/3622))
 - Move `change_role` into `group_chat_core` ([#3624](https://github.com/open-ic/open-chat/pull/3624))
+- Move `pin/unpin_message` into `group_chat_core` ([#3626](https://github.com/open-ic/open-chat/pull/3626))
```

Comment:
Perhaps all these changes could be conflated into one - it's not going to mean that much to anyone externally.